db/migrate/20160418211433_add_snapcode_image_to_developers.rb

```diff
@@ -0,0 +1,5 @@
+class AddSnapcodeImageToDevelopers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :developers, :snapcode_image, :string
```

Comment:
@siaw23 in here you're adding just one column, but the db/schema.rb below got 4 columns. In another migration you try to remove them, but they're not there so rails db:migrate crashes. Could you look into that?